.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /out-tsc
 **/tsconfig.tsbuildinfo
+infra/mountebank-imposter-config.json
```

Comment:
Wondering about the usage of mountebank. Were you mocking server responses? 
If so did you check out the [E2E testing guide about testing](https://docs.devland.is/repository/system-e2e#mocking-server-responses) ?